dandiapi/api/management/commands/create_dev_dandiset.py

```diff
@@ -30,9 +30,18 @@
 def create_dev_dandiset(*, name: str, email: str, num_extra_owners: int):
     owner = User.objects.get(email=email)
 
+    supported_licenses = [
+        license_.value for license_ in settings.DANDI_SCHEMA_INSTANCE_CONFIG.licenses
+    ]
+    supported_licenses.sort()
+
+    license_to_use = (
+        'spdx:CC0-1.0' if 'spdx:CC0-1.0' in supported_licenses else supported_licenses[0]
+    )
+
     version_metadata = {
         'description': 'An informative description',
-        'license': ['spdx:CC0-1.0'],
+        'license': [license_to_use],
```

Comment:
Why not just this (built off my other suggestion):
```suggestion
license = get_instance_config().licenses[0]
version_metadata = {
'description': 'An informative description',
        'license': license,
```